### PR TITLE
feat(mep): add commit log topics config

### PR DESCRIPTION
**Pull request by onewland**

### CODE REVIEW

**Review by evanh — APPROVED** [review]

**Comment by onewland** [comment]
blocked by https://github.com/getsentry/ops/pull/4967

**Comment by mwarkentin** [comment]
Added the topics to validation in a separate PR so that we can deploy this, then do https://github.com/getsentry/ops/pull/4967, and then finally roll out the rest of this PR.

https://github.com/getsentry/snuba/pull/2930

File: snuba/datasets/storages/generic_metrics.py [hunk 1]
```diff
@@ -119,6 +119,7 @@ def produce_policy_creator() -> DeadLetterQueuePolicy:
         processor=GenericSetsMetricsProcessor(),
         default_topic=Topic.GENERIC_METRICS,
         dead_letter_queue_policy_creator=produce_policy_creator,
+        commit_log_topic=Topic.GENERIC_METRICS_SETS_COMMIT_LOG,
```
Comment (lynnagara) [inline snuba/datasets/storages/generic_metrics.py:122]: You may want to move this line (and the distributions equivalent) into a separate PR. So we create the topics first and have them in topics.py but then start actually producing to them later.

Comment (onewland) [reply snuba/datasets/storages/generic_metrics.py]: removing that now, and then I'll merge and then we can do the ops PR

**Review by lynnagara — APPROVED** [review]

**Pull request merged**